lib/bot.js

```diff
@@ -91,6 +91,15 @@ class Bot {
       ...this.ircOptions
     };
 
+    // default encoding to UTF-8 so messages to Discord aren't corrupted
+    if (!('encoding' in ircOptions)) {
+      if (irc.canConvertEncoding()) {
+        ircOptions.encoding = 'utf-8';
+      } else {
+        logger.warn('Cannot convert message encoding; you may encounter corrupted characters with non-English text.');
```

Comment:
You might want to link to the section in your node-irc README here.

Reply:
Good idea! Done.